### DISCUSSION TITLE
pp似乎不能识别出文字的字体、大小、颜色等信息

### DISCUSSION THREAD

**Discussion by xunyao4dev** [PaddlePaddle/PaddleOCR#14264] [General]
RT，大佬们有什么方案啊？求告知

**Comment by GreatV** [comment]
这个不是ppocr的功能，这是分类任务，可以看看paddlex里面的分类模型 https://paddlepaddle.github.io/PaddleX/latest/pipeline_usage/tutorials/cv_pipelines/image_classification.html
